DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -159,6 +169,12 @@ function SetWatchFlags() {
     case "06_HONGKONG_WANCHAI_CANAL":
         WatchFlag('FoundScientistBody');
         WatchFlag('M06BoughtVersaLife');
+
+        foreach AllActors(class'FlagTrigger',fTrigger,'FoundScientist'){
+            //500 feels a bit better, so you don't have to go right into the corner
+            fTrigger.SetCollisionSize(500,CollisionHeight);
```

Comment:
The `CollisionHeight` of `self`? Should it be `fTrigger.CollisionHeight`?

Reply:
Ah, yeah - missed the fTrigger. on that one.  I see you made it taller anyway though, so all good